cibuildwheel/macos.py

```diff
@@ -535,6 +535,11 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (overwrite existing)
             if abi3_wheel is None:
+                try:
+                    os.remove(build_options.output_dir / repaired_wheel.name)
```

Comment:
Is there a reason this isn't using `Path.unlink` (perhaps with `missing_ok=True`)?

Reply:
This `missing_ok` parameter was added in Python 3.8. According to the `setup.cfg` in this repo `python_requires = >=3.6` so it still needs to run on Python 3.6.

Reply:
What about just unlink and keeping the try, then?

Reply:
Both functions do the same, so I don't really see any difference in one over the other.

Reply:
We are already using `pathlib`, so why mix that with `os`? Also this would transition more elegantly to `missing_ok` once 3.6 & 3.7 are dropped. If https://github.com/actions/setup-python/pull/411 is merged in some form, we might be able to drop 3.6 for running.

Reply:
Done. I also changed `OSError` to `FileNotFoundError` to be more specific.